src/models/TestResultsSchemaTRLCancelled.ts

```diff
@@ -21,7 +21,6 @@ const testTypesSchema = Joi.object().keys({
     ...testTypesCommonSchema,
     testTypeEndTimestamp: Joi.date().iso().required().allow(null),
     testResult: Joi.any().only(["fail", "pass", "prs", "abandoned"]).required().allow(null),
-    testExpiryDate: Joi.date().iso().allow(null),
```

Comment:
Presumably that's a yes, based on this, but can't see it mentioned anywhere on the ticket.

Reply:
And yes, it wasn't mentioned on the ticket. The ticket was quite empty. One other thing that I had to remove is the certificateNumber. This was mentioned on the front-end ticket but not on the backend.